src/geode/geosciences_io/mesh/gocad_common.cpp

```diff
@@ -58,12 +58,18 @@ namespace
                         geode::string_to_double( tokens[3] ),
                         tsurf.crs.z_sign
                             * geode::string_to_double( tokens[4] ) } } );
+                geode::detail::read_properties(
```

Comment:
maybe this could be called only if there is a 'P' at the beginning of the keyword?

Reply:
the read_properties loops on the attribute names so it won't do anything if there are no attributes, but we could add a test to check "PVRTX" if you think it's safer